lib/binaries/config_source.ts

```diff
@@ -169,8 +170,13 @@ export abstract class GithubApiConfigSource extends JsonConfigSource {
     try {
       let contents = fs.readFileSync(fileName).toString();
       let timestamp = new Date(fs.statSync(fileName).mtime).getTime();
+      let size = fs.statSync(fileName).size;
```

Comment:
`readFileSync` would throw if the file does not exist. The file exists but the file is empty. I think I could also do `let size = contents.length;`